### DISCUSSION TITLE
Top clients Naming Option

### DISCUSSION THREAD

**Discussion by nick191007** [AdguardTeam/AdGuardHome#4069] [Q&A]
Is there any possible way to name Top Clients list insted of IP. It will be easy to identify things

IP xxx.xxx.x.xxx - Test

**Comment by ainar-g** [comment]
Hello.  You can associate an IP address with a permanent client name and settings on the “Settings → Client settings” page by pressing the “Add client“ button.

**Comment by nick191007** [comment]
Thank you 👍
